rust/ql/lib/codeql/rust/dataflow/DataFlow.qll

```diff
@@ -0,0 +1,25 @@
+/**
+ * Provides a module for performing local (intra-procedural) and global
+ * (inter-procedural) data flow analyses.
+ */
+
+private import rust
+private import codeql.dataflow.DataFlow
+private import internal.DataFlowImpl as DataFlowImpl
```

Comment:
I don't think the `as DataFlowImpl` bit is needed.

Reply:
I see, that goes away with the other suggested changes 😄